server/views/components/totalAndCost.njk

```diff
@@ -1,16 +1,49 @@
-{% macro totalAndCost(field) %}
+{% macro totalAndCost(field, isFirstRow = false, isTotalRow = false) %}
     {% set totalAndCost = field.value %}
-    <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">{{ field.label }}</dt>
-        <dd class="govuk-summary-list__value">
-            <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-third">
-                    <span class="govuk-body">{{ totalAndCost.total }}</span>
+    {% if isFirstRow %}
+        <div class="govuk-summary-list__row custom-cost-header">
+            <dt class="govuk-summary-list__key"></dt>
+            <dd class="govuk-summary-list__value">
+                <div class="govuk-grid-row">
+                    <div class="govuk-grid-column-one-half">
+                        <div class="govuk-body"></div>
+                    </div>
+                    <div class="govuk-grid-column-one-half">
+                        <div><h3>Cost</h3></div>
```

Comment:
`Cost` can't be an h3 as this label has the same level of impact as the left column (`attendance time`/`preparation time`/etc). 
Making it a header will disturb the accessibility. 
=> the column `Cost` need to be a the same tag and class as the left column label.